shotover-proxy/tests/cassandra_int_tests/batch_statements.rs

```diff
@@ -43,132 +64,137 @@ async fn use_statement(connection: &CassandraConnection) {
     }
 }
 
-pub async fn test(connection: &CassandraConnection) {
-    // setup keyspace and table for the batch statement tests
-    {
-        run_query(connection, "CREATE KEYSPACE batch_keyspace WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };").await;
-        run_query(connection, "CREATE TABLE batch_keyspace.batch_table (id int PRIMARY KEY, lastname text, firstname text);").await;
+async fn insert_batch(connection: &CassandraConnection) {
```

Comment:
pulled out the restructuring changes in this file to https://github.com/shotover/shotover-proxy/pull/1144